app/views_redesigned/async_property_card_footers/show.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="row align-items-center text-center pt-4">
+  <% if @property.restrict_to_sponsor_campaigns? %>
+
+    <div class="col-4 u-ver-divider">
+      <h5 class="small font-weight-normal mb-0">
+        <span class="d-block">Total</span>
+        <span class="d-block">Clicks</span>
+      </h5>
+      <div class="text-dark mt-2"><%= number_with_delimiter @property.clicks_count(@start_date, @end_date) %></div>
+    </div>
+    <div class="col-4">
+      <h5 class="small font-weight-normal mb-0">
+        <span class="d-block">Current</span>
+        <span class="d-block">Advertiser</span>
+      </h5>
+      <div class="text-dark mt-2"><%= @property.current_sponsor_campaign&.user&.name || "none" %></div>
+    </div>
+    <div class="col-4">
+      <h5 class="small font-weight-normal mb-0">
+        <span class="d-block">Current</span>
+        <span class="d-block">Campaign</span>
+      </h5>
+      <div class="text-dark mt-2"><%= @property.current_sponsor_campaign&.name || "none" %></div>
+    </div>
+
+  <% else %>
+
+    <div class="col-4 u-ver-divider">
+      <h5 class="small font-weight-normal mb-0">
+        <span class="d-block">Total</span>
+        <span class="d-block">Impressions</span>
+      </h5>
+      <div class="text-dark mt-2"><%= number_with_delimiter @property.impressions_count(@start_date, @end_date) %></div>
+    </div>
+    <div class="col-4 u-ver-divider">
+      <h5 class="small font-weight-normal mb-0">
+        <span class="d-block">Average</span>
+        <span class="d-block">Click Rate</span>
+      </h5>
+      <div class="text-dark mt-2"><%= number_to_percentage @property.click_rate(@start_date, @end_date), precision: 2 %></div>
+    </div>
+    <div class="col-4">
+      <h5 class="small font-weight-normal mb-0">
```

Comment:
maybe just make this a `<p>` instead of adding `small` and `font-weight-normal` here.

Reply:
Same comment to the other occurrences of this in this file.

Reply:
Same questions as above.

Reply:
My opinion is we should be cleaning as much as we can (within reason). Aka trying to use consistent styling on each element and component. Along with this is creating helpers and meta classes. Ie if we use the same style on every h1, which we theoretically should, pulling those classes into our stylesheet under a new h1 class. We should be tweaking as little as possible. Hopefully that makes sense.

Reply:
Diving into this it will take some time to port. The `u-ver-divider` style is only 50 lines but relies on other scss partials. Parent and sibling divider styles, media-breakpoints, variables, etc.

We may want to backlog this for another PR when we cleanup any vendor or dangling stylesheets.